packages/react/src/components/Dropdown/DropdownSelectedItem.tsx

```diff
@@ -90,7 +90,10 @@ class DropdownSelectedItem extends UIComponent<ReactProps<DropdownSelectedItemPr
 
   componentDidUpdate(prevProps: DropdownSelectedItemProps) {
     if (!prevProps.active && this.props.active) {
+      this.itemRef.current.setAttribute('tabindex', '0')
```

Comment:
We definitely should start extracting behavior for the Dropdown component, we have too much coupling of accessibility concerns in the component... That way we would test this things separately.